library/src/main/java/com/trendyol/showcase/ui/tooltip/TooltipView.kt

```diff
@@ -72,8 +72,15 @@ class TooltipView @JvmOverloads constructor(
             with(imageViewTopArrow) {
                 visibility = tooltipViewState.getTopArrowVisibility()
                 setTint(tooltipViewState.getBackgroundColor())
-                layoutMarginStart(tooltipViewState.arrowMargin, tooltipViewState.getArrowPercentage())
-                setImageDrawable(ContextCompat.getDrawable(context, tooltipViewState.getTopArrowResource()))
+                val drawable = requireNotNull(
+                    ContextCompat.getDrawable(
+                        context,
+                        tooltipViewState.getTopArrowResource()
+                    )
+                )
+                val imageStartMargin = tooltipViewState.arrowMargin - (drawable.intrinsicWidth) / 2
```

Comment:
This operation can be done in `TooltipViewState` or `TooltipFieldUtil` like before.

Reply:
done e6fb736bec5dd4b51d46aedc39ca0d1b8731e7fa